beta/project-griffon/using-project-griffon/adobe-analytics-and-project-griffon.md

```diff
@@ -14,7 +14,7 @@ After the SDK makes a network request with Adobe Analytics, the status will tell
 | :--- | :--- |
 | `Queued` | The network request is fetching the post-processing information. |
 | `Processed` | The network request was successful, and the post-processing information is received. |
-| `Delayed` | The maximum number of requests retries to fetch the post-processing information has been exceeded. |
+| `Delayed` | The maximum number of requests to fetch the post-processing information has been exceeded. |
```

Comment:
I think this change is not needed.

Reply:
It sounds weird to me, but I'll keep it.